app/lib/utils/file.js

```diff
@@ -0,0 +1,10 @@
+// Taken from https://github.com/kevva/executable and modified
```

Comment:
Added detailed comment to explain the situation, fixed the code and opened a pull request to https://github.com/kevva/executable in order to eventually use it directly in the future.